src/domain/organization/organizationForm/merchants/merchant/Merchant.tsx

```diff
@@ -86,136 +86,113 @@ const Merchant: React.FC<Props> = ({
           name={fieldNames.active}
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelName`)}
           name={fieldNames.name}
           required
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelStreetAddress`)}
           name={fieldNames.streetAddress}
           required
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelZipcode`)}
           name={fieldNames.zipcode}
           required
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelCity`)}
           name={fieldNames.city}
           required
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelEmail`)}
           name={fieldNames.email}
           required
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelPhoneNumber`)}
           name={fieldNames.phoneNumber}
           required
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelUrl`)}
```

Comment:
Merchants do not have a user-editable `url` field anymore in the back-end. The field can be removed from the front-end.

Reply:
Removed url from the merchant fields